Makefile

```diff
@@ -56,7 +56,9 @@ endif
 
 # Everything else...
 
-NAME ?= emissary
+EMISSARY_NAME ?= emissary
+LCNAME := $(shell echo $(EMISSARY_NAME) | tr '[:upper:]' '[:lower:]')
```

Comment:
you might be able to remove LCNAME here because it will be imported from the below include statement. It appears its first usage in this file is after the imports.

Reply:
you might also need to swap
```
include $(OSS_HOME)/build-aux/check.mk
include $(OSS_HOME)/build-aux/builder.mk
```
to make that work